src/analysis/special_functions/gamma.lean

```diff
@@ -928,4 +954,82 @@ end
 
 end bohr_mollerup
 
+section strict_mono
+
+lemma Gamma_two : Gamma 2 = 1 := by simpa using Gamma_nat_eq_factorial 1
+
+lemma Gamma_three_div_two_lt_one : Gamma (3 / 2) < 1 :=
+begin
+  -- This can also be proved using the closed-form evaluation of `Gamma (1 / 2)` in
+  -- `analysis.special_functions.gaussian`, but we give a self-contained proof using log-convexity
+  -- to avoid unnecessary imports.
+  have A : (0:ℝ) < 3/2, by norm_num,
+  have := bohr_mollerup.f_add_nat_le convex_on_log_Gamma (λ y hy, _) two_ne_zero one_half_pos
+    (by norm_num : 1/2 ≤ (1:ℝ)),
+  swap, { rw [function.comp_app, Gamma_add_one hy.ne', log_mul hy.ne' (Gamma_pos_of_pos hy).ne',
+    add_comm] },
+  rw [function.comp_app, function.comp_app, nat.cast_two, Gamma_two, log_one, zero_add,
+    (by norm_num : (2:ℝ) + 1/2 = 3/2 + 1), Gamma_add_one A.ne',
+    log_mul A.ne' (Gamma_pos_of_pos A).ne', ←le_sub_iff_add_le',
+    log_le_iff_le_exp (Gamma_pos_of_pos A)] at this,
+  refine this.trans_lt (exp_lt_one_iff.mpr _),
+  rw [mul_comm, ←mul_div_assoc, div_sub' _ _ (2:ℝ) two_ne_zero],
+  refine div_neg_of_neg_of_pos _ two_pos,
+  rw [sub_neg, mul_one, ←nat.cast_two, ←log_pow, ←exp_lt_exp, nat.cast_two, exp_log two_pos,
+    exp_log];
+  norm_num,
+end
+
+lemma Gamma_gt_one_of_gt_two {x : ℝ} (hx : 2 < x) : 1 < Gamma x :=
+begin
```

Comment:
This lemma and the next one are special instances of the following fact: if `f` is convex on an interval containing two points `x` and `y` with `f x < f y`, then `f` is strictly increasing on the interval `[y, ...)`. Could you instead prove the general fact, and deduce the specific case of the Gamma function from it, to avoid all computations specific to Gamma?

Reply:
Fair point. I had a go at redoing this; and in the process, realised that a bunch of lemmas in `strict_convex` (`strict_convex_on.lt_left_of_right_lt'` and its seven variants) are all true under the weaker assumption of convexity, rather than strict convexity. This'll take a bit of rewriting; I'll check something in when I get the time.

Reply:
Easier than I thought: the proofs generalised immediately, and those eight lemmas weren't used anywhere in the library so there was no fallout to deal with.